### PR TITLE
 project

### LINKED ISSUE #2: Middle Section - Portfolio
**Make a similar middle section as shown in the screenshot**

Screenshot 2023-05-13 at 17 55 19

Screenshot 2023-05-13 at 17 57 31

### PR DESCRIPTION
**Pull request by Pranjal8989**
Screenshot (26)
Screenshot (27)

### CODE REVIEW

**Comment by Pranjal8989** [comment]
@MohitGupta121 PTAL. thanks

File: project.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (MohitGupta121) [inline project.html]: Change this file to index.html

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@Pranjal8989 PTAL, Please make changes and colors as per screenshot, 
Thanks.

**Comment by Pranjal8989** [comment]
@MohitGupta121 PTAL . thanks

**Pull request closed without merging**